addons/mail/static/src/core/common/composer.js

```diff
@@ -503,11 +503,11 @@ export class Composer extends Component {
     }
 
     async onClickFullComposer(ev) {
+        const allRecipients = [...this.thread.suggestedRecipients];
         if (this.props.type !== "note") {
-            // auto-create partners of checked suggested partners
-            const newPartners = this.thread.suggestedRecipients.filter(
-                (recipient) => recipient.checked && !recipient.persona
-            );
+            allRecipients.push(...this.thread.additionalRecipients);
```

Comment:
Same question as before, is `additionalRecipients` mutually exclusive with `suggestedRecipients` ?